apps/condo/domains/organization/utils/serverSchema/Organization.js

```diff
@@ -29,32 +29,79 @@ async function updateOrganizationEmployee (context, id, data) {
+
+async function createDefaultRoles (context, organization, data) {
     if (!context) throw new Error('no context')
     if (!organization.id) throw new Error('wrong organization.id argument')
     if (!organization.country) throw new Error('wrong organization.country argument')
-    const adminRoleName = (COUNTRIES[organization.country] || COUNTRIES[DEFAULT_ENGLISH_COUNTRY]).adminRoleName
-    return await execGqlWithoutAccess(context, {
-        query: OrganizationEmployeeRole.CREATE_OBJ_MUTATION,
-        variables: {
-            data: {
-                organization: { connect: { id: organization.id } },
-                canManageOrganization: true,
-                canManageEmployees: true,
-                canManageRoles: true,
-                canManageIntegrations: true,
-                canManageProperties: true,
-                canManageTickets: true,
-                canManageContacts: true,
-                name: adminRoleName,
-                ...data,
-            },
+    const langDict = COUNTRIES[organization.country] || COUNTRIES[DEFAULT_ENGLISH_COUNTRY]
+    // TODO: place to another file?
+    const defaultRoles = {
+        admin: {
+            canManageOrganization: true,
+            canManageEmployees: true,
+            canManageRoles: true,
+            canManageIntegrations: true,
+            canManageProperties: true,
+            canManageTickets: true,
+            canManageContacts: true,
         },
-        errorMessage: '[error] Create admin role internal error',
-        dataPath: 'obj',
+        supervisor: {
+            canManageOrganization: false,
+            canManageEmployees: false,
+            canManageRoles: false,
+            canManageIntegrations: false,
+            canManageProperties: true,
+            canManageTickets: true,
+            canManageContacts: true,
+        },
+        manager: {
+            canManageOrganization: false,
+            canManageEmployees: false,
+            canManageRoles: false,
+            canManageIntegrations: false,
+            canManageProperties: true,
+            canManageTickets: true,
+            canManageContacts: true,
+        },
+        foreman: {
+            canManageOrganization: false,
+            canManageEmployees: false,
+            canManageRoles: false,
+            canManageIntegrations: false,
+            canManageProperties: false,
+            canManageTickets: false,
+            canManageContacts: false,
+        },
+        technician: {
+            canManageOrganization: false,
+            canManageEmployees: false,
+            canManageRoles: false,
+            canManageIntegrations: false,
+            canManageProperties: false,
+            canManageTickets: false,
+            canManageContacts: false,
+        },
+    }
+    Object.keys(defaultRoles).forEach((roleId) => {
+        defaultRoles[roleId].name = langDict.roleNames[roleId]
     })
+    const tasks = Object.values(defaultRoles).map((roleInfo) => 
```

Comment:
why can't we use simple server utils
```
OrganizationEmployeeRole.create
```
here?

Reply:
It's better to use `OrganizationEmployeeRole.create` here